docs/NEO-EDEN.md

```diff
@@ -0,0 +1,199 @@
+# NeoEden: framework that allows you to write EVE tests in Golang
```

Comment:
This is a really big improvement. I am going to put some comments and suggestions inline.

The main one, though, is to separate the friendly and fun, "what is neoeden and why do we need it and what was the history" (first ~92 lines), which will become more of a state-in-time curiosity once everything else is converted, from a "how to test with eden" doc (line 93 to end), which will continue to be up to date.